UI/Marker/EnemyPlacemark.gd

```diff
@@ -11,9 +11,11 @@ onready var _detailsPanel: DetailsPanel = DetailsPanel.get_instance(get_tree())
 
 func _ready():
 	# Connect to the custom signal emitted by SetProvider
-	var set_provider = get_node("../../../../../../SetProvider")
+	var set_provider = get_node("../../../../../../root/ui/status_view/container/DayNightOptionButton")
 	if set_provider:
```

Comment:
No need to do look up the node by the path when you can do SetProvider.connect("day_night_selected", self, "_on_day_night_selected")